public/css/style.css

```diff
@@ -28,6 +28,13 @@
     height: 150px;
 }
 
+.selected-image-display {
+    margin: 1px 1px -3px 1px;
+    padding: 0 0 0 0;
+    width: 300px;
+    height: 300px;
+}
+
```

Comment:
Working with CSS tends to be a combination of knowledge, trial and error and stack overflow.. :P There's plenty of great UI texts out there, but there's no authority on CSS. 

Yes, absolutely -- responsiveness is something to think about. On my 24" monitor at home, there's lots of wasted screen real estate when I ruin the UI. Once we know exactly how our pages will be laid out, I'd like to work in % too. Once we switch over to node-web-kit, it's possible to specify a minimum resolution for the window. So we could work on that minimum resolution, then just ensure it resizes well to larger screens.